api.md

```diff
@@ -1669,14 +1669,15 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
          [ optional grouping exprs>]
 [HAVING ...]
 ```
+Note that a single hypertable is supported in the `FROM` clause.
```

Comment:
```suggestion
Note that continuous aggregates have some limitations of what types of
queries they can support, described in more length below.  For example,
the `FROM` clause must provide a single hypertable name (no JOINs or subqueries),
the `GROUP BY` must include a time bucket on the hypertable's time column, and
all aggregates must be parallelizable.
```

Reply:
Used the above text with some modifications.